packages/next/compiled/babel-packages/package.json

```diff
@@ -1 +1 @@
-{"name":"@babel/core","main":"packages-bundle.js","author":"The Babel Team (https://babel.dev/team)","license":"MIT"}
+{"name":"@babel/preset-env","main":"packages-bundle.js","author":"The Babel Team (https://babel.dev/team)","license":"MIT"}
```

Comment:
Is this expected to be `@babel/preset-env`?

Reply:
Yeah we just need to use some valid package name so I picked one of the packages included in this bundle